plugins/catalog-backend/src/modules/core/AnnotateLocationEntityProcessor.ts

```diff
@@ -53,12 +53,16 @@ export class AnnotateLocationEntityProcessor implements CatalogProcessor {
     let viewUrl;
     let editUrl;
     let sourceLocation;
+    const gitCommitBranchURLPattern = /\b[0-9a-f]{5,40}\b/;
```

Comment:
unfortunately this will also match branches

Reply:
yes right now testing this 
const commitHashRegex = /\/blob\/[0-9a-f]{40}\//;

Reply:
I'm thinking `{40,}` could be good? It's match both sha1 and future formats. We won't even have commit hash shorthands in the URL right? At least doesn't seem like something we have to support